mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessageTest.java

```diff
@@ -61,7 +61,7 @@
     public static final MessageUid MESSAGE_UID = MessageUid.of(154);
 
     @ClassRule
```

Comment:
actually, you don't need a ClassRule here at all :)
```
public static TikaContainer tika = new TikaContainer();
static {
   tika.start();
}
```

will give you the same result :)

Reply:
Hi!

First thanks for the feedback.

We did discuss that in detail out here https://github.com/linagora/james-project/pull/1584#discussion_r205094601 . The outcome is that we want to hide the fact that we are using a singleton under the hood, and ease future migration back to JUNIT4 rules.